### PR TITLE
kind.sh: Make kind image configurable

### PR DESCRIPTION
**Pull request by flavio-fernandes**
Trivial: This is particularly helpful when working around pull rate limit
issues. Update documentation for kind.sh

Signed-off-by: Flavio Fernandes <flaviof@redhat.com>

### CODE REVIEW

**Comment by tssurya** [comment]
/lgtm

The default is still `kindest/node`,  making it configurable will allow us to choose other versions.

File: docs/kind.md [hunk 1]
```diff
@@ -418,6 +418,16 @@ Once testing is complete, to tear down the KIND deployment:
 $ kind delete cluster --name ovn
 ```
 
+### Using specific Kind container image and tag
```
Comment (astoycos) [inline docs/kind.md:421]: We should probably have some warning here like "you may break everything if you change the node image" :)

Comment (flavio-fernandes) [reply docs/kind.md:421]: done. ;)

**Review by trozet — APPROVED** [review]

**Pull request merged**